projects/netgrif-components/src/lib/navigation/group-navigation-component-resolver/default-components/simple-views/default-simple-case-view/default-simple-case-view.component.html

```
@@ -0,0 +1,35 @@
+<div class="case-view-container" fxLayout="column" fxLayoutAlign="start stretch">
+    <nc-breadcrumbs></nc-breadcrumbs>
+
+    <div class="case-view-search-container">
+        <div fxLayout="row" fxLayoutAlign="space-between">
+            <div fxLayoutAlign="start center" fxFlex>
+                <nc-search class="search-width" *ngIf="(enableSearch)" [showSearchToggleButton]="showToggleButton"
+                           [initialSearchMode]="initialSearchMode"></nc-search>
+            </div>
+            <div fxLayoutAlign="end center" *ngIf="allowExport">
+                <button mat-mini-fab color="primary" class="export-mat-mini-fab" (click)="export()"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)